lunar/distribution.yaml

```diff
@@ -238,11 +238,20 @@ repositories:
       version: kinetic-devel
     status: maintained
   catkin_pip:
+    doc:
+      type: git
+      url: https://github.com/pyros-dev/catkin_pip.git
+      version: devel
     release:
       tags:
         release: release/lunar/{package}/{version}
       url: https://github.com/pyros-dev/catkin_pip-release.git
-      version: 0.2.2-0
+      version: 0.2.3-0
+    source:
+      test_pull_requests: true
```

Comment:
> webhook added.

:+1:

I'll leave it to @dirk-thomas to answer to https://github.com/ros/rosdistro/pull/15762#discussion_r132618848